media/windows/ECC-Ecobee/requirements.txt

```diff
@@ -1,7 +1,16 @@
+datetime
+time
 pythonping
 dnspython
+socket
 certifi
 pytz
+json
+os
+fnmatch
+sys
```

Comment:
FWIW, sys, os, json, time, and datetime are all core Python modules (i.e., they're guaranteed to be present).  I don't know offhand which of the others are also core (socket *might* be...?).

Reply:
Okay, I assumed some of these were but didn't take the time to track which ones were / not in the core.  Probably should have researched that first but included them just in case.